### ISSUE TITLE
MissingPluginException (MissingPluginException(No implementation found for method setLogLevel on channel plugins.pauldemarco.com/flutter_blue/methods))

### ISSUE THREAD

**Issue by ramsestom** [pauldemarco/flutter_blue#532]
Have this error when trying to run the example app on an android 5.1 device but this do not appear when on an android 8.0 device.
More details on this thread: https://github.com/pauldemarco/flutter_blue/issues/530 that its author has closed while the problem persists...

**Comment by pauldemarco** [comment]
This is typically only seen in the emulator. Just to clarify, you are seeing this on an physical Android device? Can you share the model number?

**Comment by ramsestom** [comment]
Yes I am seeing this on a physical Android device. This is a Xiaomi Redmi 3: https://www.gsmarena.com/xiaomi_redmi_3-7862.php with Android 5.1.1

**Comment by SimeonVSimeonov** [comment]
And I seeing this too on a physical Android device. Huawei Y6II Compact : https://www.gsmarena.com/huawei_y6ii_compact-8663.php with Android 5.1.1

**Comment by yageorge** [comment]
Same error on an old device HTC M8

**Comment by GuylianStevens1** [comment]
Can the package be used while debugging with the Chrome extension. As I do to receive this same error?
